Speedruns.Backend/Repositories/ConsolesRepository.cs

```diff
@@ -9,7 +9,7 @@ public class ConsolesRepository : IConsolesRepository
         private readonly DbSet<ConsoleEntity> _consoles;
         public ConsolesRepository(SpeedrunsContext context)
         {
-            _consoles = context.Consoles;
+            _consoles = context.Set<ConsoleEntity>();
```

Comment:
In the `DbContextMock`, it's mocking out the `Set` method on the `DbContext`, so the repositories will need to access their `DbSet` this way instead to ensure the tests make use of that mocked method (you cannot mock the `DbSet` properties.

`_context.Consoles` is the same as `_context.Set<ConsoleEntity>()`; they will both yield the `DbSet<ConsoleEntity>`.